src/websocket/server.cc

```diff
@@ -183,23 +267,65 @@ future<> connection::read_loop() {
         });
     }).then_wrapped([this] (future<> f) {
         if (f.failed()) {
-            wlogger.error("Read failed: {}", f.get_exception());
+            wlogger.error("Failure: {}", f.get_exception());
         }
         return _replies.push_eventually({});
     }).finally([this] {
         return _read_buf.close();
     });
 }
 
+future<> connection::send_data(temporary_buffer<char> buff) {
+    sstring data;
+    data.append("\x81", 1);
```

Comment:
nitpick: can't you just create the string with this initial value? Why create it empty and then append?